internal/resources/metal/organization_member/resource.go

```diff
@@ -0,0 +1,298 @@
+	return &Resource{
+		BaseResource: framework.NewBaseResource(
+			framework.BaseResourceConfig{
+				Name:   "equinix_metal_organization_member",
+				Schema: GetResourceSchema(),
+			},
+		),
+	}
+}
+
+type Resource struct {
+	framework.BaseResource
+}
+
+func (r *Resource) ImportState(
+	ctx context.Context,
+	req resource.ImportStateRequest,
+	resp *resource.ImportStateResponse,
+) {
+	tflog.Debug(ctx, "importer Organization")
+
+	parts := strings.Split(req.ID, ":")
+	if len(parts) != 2 {
+		return
+
+	}
+	invitee := parts[0]
+	orgID := parts[1]
+
+	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, tfpath.Root("invitee"), invitee)...)
+	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, tfpath.Root("organization_id"), orgID)...)
+}
+
+func (r *Resource) Create(
+	ctx context.Context,
+	req resource.CreateRequest,
+	resp *resource.CreateResponse,
+) {
+	r.Meta.AddFwModuleToMetalUserAgent(ctx, req.ProviderMeta)
+	client := r.Meta.Metal
+
+	var plan ResourceModel
+	resp.Diagnostics.Append(req.Plan.Get(ctx, &plan)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	invite := plan.Invitee.ValueString()
+
+	roles := make([]string, 0)
+	for _, elem := range plan.Roles.Elements() {
+		if strValue, ok := elem.(types.String); ok {
+
+			if !strValue.IsNull() {
+				roles = append(roles, strValue.ValueString())
+			}
+		}
+	}
+
+	projects := make([]string, 0)
+	for _, elem := range plan.ProjectsIDs.Elements() {
+		if strValue, ok := elem.(types.String); ok {
+			projects = append(projects, strValue.ValueString())
+		}
+	}
+
+	createRequest := &packngo.InvitationCreateRequest{
+		Invitee:     invite,
+		Roles:       roles,
+		ProjectsIDs: projects,
+		Message:     strings.TrimSpace(plan.Message.ValueString()),
+	}
+
+	orgID := plan.OrganizationID.ValueString()
+	_, _, err := client.Invitations.Create(orgID, createRequest, nil)
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Failed to create Organizations",
+			equinix_errors.FriendlyError(err).Error(),
+		)
+		return
+	}
+
+	listOpts := &packngo.ListOptions{Includes: []string{"user"}}
+	invitations, _, err := client.Invitations.List(orgID, listOpts)
+	if err != nil {
+		err = equinix_errors.FriendlyError(err)
+		// If the org was destroyed, mark as gone.
+		if equinix_errors.IsNotFound(err) {
+			plan.OrganizationID = basetypes.NewStringNull()
+			return
+		}
+		return
+	}
+
+	members, _, err := client.Members.List(orgID, listOpts)
+	if err != nil {
+		err = equinix_errors.FriendlyError(err)
+		// If the org was destroyed, mark as gone.
+		if equinix_errors.IsNotFound(err) {
+			return
+		}
+		return
+	}
+
+	member, err := findMember(invite, members, invitations)
+	if err != nil {
+		log.Printf("[WARN] Could not find member %s in organization, removing from state", plan.OrganizationID)
+		plan.OrganizationID = basetypes.NewStringNull()
+		return
+	}
+	// Parse API response into the Terraform state
+	if member != nil {
+		diags := plan.parse(ctx, member)
+		if diags.HasError() {
+			return
```

Comment:
Probably need to add the error to `resp.Diagnostics` here.
```suggestion
			resp.Diagnostics.Append(diags)
			return
```